src/client/datascience/data-viewing/types.ts

```diff
@@ -68,6 +68,8 @@ export interface IDataFrameInfo {
     maximumRowChunkSize?: number;
     type?: string;
     originalVariableType?: string;
+    name?: string;
+    fileName?: string;
```

Comment:
Would be nice to have come comments. What's `FileName` here?
The rest seem to make sense, as they are associated with a variable.